proposals/required-members.md

```diff
@@ -333,72 +337,21 @@ generated. If the constructor already has an `Obsolete` attribute, no additional
 members, the generated `Obsolete` attribute will reference the `required` error message, under the assumption that if a compiler is new enough to
 understand `required` properties, it will also understand by-ref-like types.
 
-To build the full list of `required` members for a given type `T`, including all base types, all the `RequiredMembersAttribute`s from `T` and its
-base type `Tb` (recursively until reaching `System.Object`) are collected. At every `Ti`, if it has a `RequiredMembersAttribute` with members `R1`..`Rn`,
+**TODO**: LDM has indicated that we desire some kind of dedicated poison attribute, specifically for compilers to indicate that a component is not
+usable in downlevel compilers without resorting to a modreq (ie, a binary breaking change). This will need to be designed and applied to the constructor
+in addition to Obsolete.
+
+To build the full list of `required` members for a given type `T`, including all base types, all the `RequiredMemberAttribute`s from `T` and its
+base type `Tb` (recursively until reaching `System.Object`) are collected. At every `Ti`, if it has a `RequiredMemberAttribute` with members `R1`..`Rn`,
 the following algorithm is performed:
 
 1. Perform standard member lookup on `Ti` for name `Ri` with 0 arguments.
```

Comment:
nit: is this really a lookup or better described as simply enumerating the members?

Reply:
I meant to update this section entirely, will do so in the morning.